package.json

```diff
@@ -23,7 +23,9 @@
     "predocs-test": "yarn docs",
     "docs-test": "lerna run docs-test",
     "docs": "lerna run docs",
-    "docs-deploy": "gh-pages --dist packages/opentelemetry-types/docs/out"
+    "docs-deploy": "gh-pages --dist packages/opentelemetry-types/docs/out",
+    "lint-examples": "node_modules/.bin/eslint ./examples/*/**.js",
```

Comment:
you don't need to explicitly put `node_modules/.bin/eslint`, when run as part of an npm script this is done automatically.

Reply:
You have your globbing backwards. It should be `./examples/**/*.js`

Reply:
Oh good to know and oops yes thank you.